Solution.yaml

```diff
@@ -502,6 +502,17 @@ parameters:
         type: cypher
         query: 'MATCH(n:Customer) RETURN n.id as id'
         resultKey: id
+  - id: 'scenario_to_compare_1'
+    labels:
+      fr: 'Scénario à comparer'
+      en: 'Scenario to compare'
+    varType: 'enum'
+    options:
+      subType: 'SCENARIOS'
+      runTemplateFilter: ['1', '3']
+      tooltipText:
+        fr: "Scénario auquel on va comparer les données"
+        en: "Scenario to which compare data"
```

Comment:
Better phrasing would be _Scenario **against** which **to** compare data_ :slightly_smiling_face: